pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -0,0 +1,270 @@
+package exporter
+
+import (
+	"fmt"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter/ipfix"
+	"hash/fnv"
+	"net"
+	"os"
+	"strings"
+	"time"
+	"unicode"
+
+	ipfixentities "github.com/srikartati/go-ipfixlib/pkg/entities"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter/flowrecords"
+)
+
+var (
+	IANAInfoElements = []string{
+		"flowStartSeconds",
+		"flowEndSeconds",
+		"sourceIPv4Address",
+		"destinationIPv4Address",
+		"sourceTransportPort",
+		"destinationTransportPort",
+		"protocolIdentifier",
+		"packetTotalCount",
+		"octetTotalCount",
+		"packetDeltaCount",
+		"octetDeltaCount",
+		"reverse_PacketTotalCount",
+		"reverse_OctetTotalCount",
+		"reverse_PacketDeltaCount",
+		"reverse_OctetDeltaCount",
+	}
+	AntreaInfoElements = []string{
+		"sourcePodName",
+		"sourcePodNamespace",
+		"destinationPodName",
+		"destinationPodNamespace",
+	}
+)
+
+var _ FlowExporter = new(flowExporter)
+
+type FlowExporter interface {
+	Run(stopCh <-chan struct{})
+}
+
+type flowExporter struct {
+	flowRecords  flowrecords.FlowRecords
+	process      ipfix.IPFIXExportingProcess
+	elementsList []*ipfixentities.InfoElement
+	templateID   uint16
+}
+
+func getNodeName() (string, error) {
+	const nodeNameEnvKey = "NODE_NAME"
```

Comment:
Probably let us pass the Node name from the Agent initialization code. Would not scatter the logic of reading from env variable everywhere.

Reply:
May be I could use pkg/util/env code? Did not realize that utility func was available.

Reply:
pkg/util/env sounds good.